spring-petclinic-microservices/spring-petclinic-config-server/pom.xml

```diff
@@ -7,6 +7,7 @@
 	<artifactId>spring-petclinic-config-server</artifactId>
 	<packaging>jar</packaging>
 	<description>Spring PetClinic Config Server</description>
+    <name>Spring Petclinic Config Server</name>
```

Comment:
Indent problem.

Reply:
it should be fixed already;
![image](https://user-images.githubusercontent.com/4465723/148519528-7ec3261b-1118-491e-bb31-cf93d8d42674.png)

https://github.com/Azure-Samples/azure-spring-boot-samples/blob/f3ee0fec0fd9ad5eacc5fa4fea45cbac0c52581c/spring-petclinic-microservices/spring-petclinic-config-server/pom.xml